examples/mysql-atomic-batch-execute-operation/mysql_atomic_batch_execute_operation.bal

```diff
@@ -36,70 +37,67 @@ public function main() returns error? {
     ];
 
     // Creates a batch parameterized query.
-    sql:ParameterizedQuery[] insertQueries = 
-        from var data in insertRecords
+    sql:ParameterizedQuery[] insertQueries =
+        from var customer in customers
         select `INSERT INTO Customers
                 (firstName, lastName, registrationID, creditLimit, country)
-                VALUES (${data.firstName}, ${data.lastName},
-                ${data.registrationID}, ${data.creditLimit}, ${data.country})`;
+                VALUES (${customer.firstName}, ${customer.lastName},
+                ${customer.registrationID}, ${customer.creditLimit}, 
+                ${customer.country})`;
 
     // The transaction block can be used to roll back if any error occurred.
     transaction {
-        var result = mysqlClient->batchExecute(insertQueries);
+        sql:ExecutionResult[]|sql:Error result = 
+                                    mysqlClient->batchExecute(insertQueries);
```

Comment:
Shall we have two simple execute instead of batch executes to explain transaction support?

Reply:
Changed in [5eb5d05](https://github.com/ballerina-platform/ballerina-distribution/pull/2842/commits/5eb5d051a49207a611ceb0faf1d373d364965b4f)